lib/std/posix.zig

```diff
@@ -4817,7 +4817,7 @@ pub const AccessError = error{
 ///
 /// On Windows, `mode` is ignored. This is a POSIX API that is only partially supported by
 /// Windows. See `fs` for the cross-platform file system API.
-pub fn access(path: []const u8, mode: u32) AccessError!void {
+pub fn access(path: []const u8, mode: c_int) AccessError!void {
```

Comment:
Andrew prefers to keep `c_int` (& co) usage limited to `std.c`. It's unclear to me where that leaves `std.posix`, considering POSIX is an extension of C. cc @andrewrk

Reply:
Please try to think of "std.posix" as a concept that is what it needs to be rather than being defined by some third party group. Everyone gets hung up on this, so I think it will have to be renamed yet again.

POSIX is irrelevant. C is irrelevant. This is an abstraction layer

Reply:
So just to be completely clear: no `c_int`, yeah?

Reply:
yeah, no c_int in std.posix function definitions